### ISSUE TITLE
Update the project Lando, DDEV and Drupal to latest - release 2.0

### ISSUE THREAD

**Issue by hkirsman** [aleksijohansson/drupal-project#234]
Currently this project is still using PHP 7.4 and Drupal 9.1

@aleksijohansson 

## Todo

- Create new branch after release 1.0
- Cherry-pick commits from this issue
- Make sure Lando and DDEV are running on similar setup and also check Drupal status page. DDEV did have some warning about APCu cache size.
- Agree if we'll stay on Druapal 10 or go to Drupal 11

**Comment by aleksijohansson** [comment]
Agreed, there is definitely some updating to do. Feel free to work on it @hkirsman if you have time.

**Comment by hkirsman** [comment]
Will do. In theory delete drupal project files, copy over new files from that project. Our customisation is the benchmark script and maybe something in Lando/DDEV

**Comment by aleksijohansson** [comment]
Sounds about right. There is also the basic installation profile, that might need updating.

**Comment by hkirsman** [comment]
I've created 2 commits, but they sit still in a temporary branch. After we current open PR's and close https://github.com/aleksijohansson/drupal-benchmark/issues/327 , I'll create new branch and cherry-pick these commits.

I took the Lando setup before they moved to Drupal 11 from here https://github.com/wunderio/drupal-project/commit/fcbf8e404ab8eb9b516d42a537bc1a1acdf64f9b

Should we also go tot Drupal 11 with this upgrade?


@aleksijohansson

**Comment by aleksijohansson** [comment]
Going to Drupal 11 with this upgrade makes sense to me 👍
